libs/error-analysis/src/lib/ErrorAnalysisDashboard/ErrorAnalysisDashboard.tsx

```diff
@@ -805,7 +812,8 @@ export class ErrorAnalysisDashboard extends React.PureComponent<
     this.setState((prevState) => ({
       baseCohort: switchNew ? savedCohort : prevState.baseCohort,
       cohorts: newCohorts,
-      selectedCohort: switchNew ? savedCohort : prevState.selectedCohort
+      selectedCohort: switchNew ? savedCohort : prevState.selectedCohort,
+      showMessageBar: true
```

Comment:
why we don't need to check if it is switchNew or not? just for my understanding

Reply:
Ah, so you can save a cohort but not switch to it.  If you just save and don't switch, we keep the current base/selected cohorts the same.  If you save and switch to the new one, then we set the base and selected cohort to be the new one.